environment.yml

```
@@ -1,11 +1,15 @@
-name: workshop
+name: image-analysis-23
 channels:
   - conda-forge
 dependencies:
+  - python=3.10
+  - pip
   - jupyterlab
   - jupytext
-  - napari
-  - pip
+  - matplotlib
+  - napari=0.4.17
+  - scikit-image=0.20.0
+  - jupyter-book=0.15.1
+  - napari-animation
   - pip:
```

```suggestion
  - pooch==1.7.0
  - scikit-learn==1.2.2
  - pip:
```
Would be nice to have access to scikit-image datasets and full functionality for segmentation, not sure if we will need this though (see https://github.com/scikit-image/skimage-tutorials/blob/69f245cbba2f9b0a2f07d7e44a861d9bc2179656/requirements.txt)